server/db.js

```diff
@@ -0,0 +1,22 @@
+const {Client} = require('pg');
+
+const client = new Client({
+  database: 'megabites',
+  user: 'jane',
+});
+
+client.connect()

```

Comment:
I am unsure whether exporting the client within the promise is necessary

Reply:
I wasn't sure either! I was thinking that I should only allow access if the connection was successful, but I suppose it will error out no matter what...so I can put it somewhere neater : )